src/app/grants/developer/overview.tsx

```diff
@@ -0,0 +1,58 @@
+import ActivitiesNewsletter from "@/components/ActivitiesNewsLetterFooter";
+import { useEffect, useRef } from "react";
+import Cohort from "../cohort";
+import FaqContainer from "../faq";
+import StartGrant from "../starter";
+import DeveloperInfo from "./info";
+import OverviewHero from "./overviewHero";
+
+export default function OverviewDeveloper({
+  setSection,
+  setPage,
+}: {
+  setSection: (section: string) => void;
+  setPage: (page: string) => void;
+}) {
+  const firstDivRef = useRef<HTMLDivElement | null>(null);
+
+  useEffect(() => {
+    if (firstDivRef.current) {
+      firstDivRef.current.scrollIntoView({ behavior: "smooth" });
+      firstDivRef.current.focus();
+    }
+  }, []);
+  return (
+    <div ref={firstDivRef} className="flex flex-col w-full">
+      <OverviewHero setPage={setPage} />
+      <DeveloperInfo />
+      <StartGrant setPage={setSection} />
+      <Cohort setPage={setSection} />
+      <FaqContainer
+        faqs={[
+          {
+            answer: "We offer two types of developer grants:",
+            question: "What types of Developer Grants does Btrust offer?",
+            lists: [
+              "Long-Term Grants (BOSC) – For developers making sustained contributions to Bitcoin open-source projects.",
```

Comment:
Just so we don't confuse:

```suggestion
              "Btrust Open-Source Cohort (BOSC) – Long-Term grants for developers making sustained contributions to Bitcoin open-source projects.",
```